Libraries/Azure.psm1

```diff
@@ -505,25 +510,42 @@ Function Create-ResourceGroup([string]$RGName, $location, $CurrentTestData) {
     return $retValue
 }
 
-Function Create-ResourceGroupDeployment([string]$RGName, $location, $TemplateFile, $UseExistingRG, $ResourceCleanup) {
-    $FailCounter = 0
-    $retValue = "False"
+Function Create-ResourceGroupDeployment([string]$RGName, $TemplateFile, $UseExistingRG, $ResourceCleanup) {
+    $retValue = $false
```

Comment:
please initialize 2 variables at the same place.

Reply:
can you please mention the name of the 2 variables? now it s just one.

Reply:
i believe @juhlee-microsoft is referring to the `FailCounter` variable that was removed

Reply:
@juhlee-microsoft please resolve this issue if you think it has been addressed. thanks.

Reply:
$retValue  & $errMsg

Reply:
done